tests/Browser/LoginTest.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace Tests\Browser;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use App\User;
+use Hash;
+
+class LoginTest extends DuskTestCase
+{
+    /**
+     * A basic browser url not login.
+     * case 1-1
+     */
+    public function testPathWrong()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/abc')
+                    ->assertSee('404');
+        });
+    }
+
+    /**
+     * A basic browser go to path admin dislay Login.
+     * Case 1-2
+     */
+    public function testPathAdmin()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit(config('contants.path.admin'))
+                    ->assertPathIs(config('contants.path.login'));
+        });
+    }
+
+    /**
+     * A basic browser go to Forget passord.
+     * case 1-3
+     */
+    public function testDisplayLogin()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit(config('contants.path.login'))
+                    ->assertSee('Login');
+        });
+    }
+
+
+    /**
+     * Display login test forget password link.
+     * Case 1-4
+     */
+    public function testForgetPasswordLink()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit(config('contants.path.login'))
+                    ->clickLink('Forgot Your Password?')
+                    ->assertPathIs('/password/reset');
+        });
+    }
+
+    /**
+     * Display login test register link.
+     * Case 1-5
+     */
+    public function testRegisterLink()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit(config('contants.path.admin'))
+                    ->clickLink('Register')
+                    ->assertPathIs('/register');
+        });
+    }
+
+    /**
+     * Test login not input.
+     * Case 1-6
+     */
+    public function testLoginNotInput()
+    {
+        $this->browse(function ($browser) {
+            $browser->visit(config('contants.path.login'))
+                    ->press('Login')
```

Comment:
Test case not have press Login . which once correct ?

Reply:
i fixed in report testcase